pkgs/development/compilers/pforth/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "28";
+  name = "pforth";
+  src = fetchFromGitHub {
+    owner = "philburk";
+    repo = "pforth";
```

Comment:
```suggestion
    repo = pname;
```

Reply:
I don't think this is a good idea. The repository does not depend on how we call the package.

Reply:
it was just to avoid duplicating the `pname`, but it's not a blocker to merging :)